apps/demo/src/app/checkout/components/place-order/place-order.component.ts

```diff
@@ -19,11 +18,14 @@ export class PlaceOrderComponent implements OnInit {
   enablePlaceOrderButton$: Observable<boolean>;
   cart$: Observable<DaffCart>;
 
-  constructor(private store: Store<fromCart.State>) { }
+  constructor(
+		private store: Store<any>,
+		private cartFacade: DaffCartFacade<DaffCart>
+	) { }
 
   ngOnInit() {
     this.enablePlaceOrderButton$ = this.store.pipe(select(fromDemoCheckout.selectEnablePlaceOrderButton));
-    this.cart$ = this.store.pipe(select(fromCart.selectCartValue));
+    this.cart$ = this.cartFacade.cart$;
   }
 
   placeOrder() {
```

Comment:
Can you dispatch with the facade here so you can remove the store from this component?

Reply:
I can't remove the dependency on store for this component, because it references demo store and not just daffodil store. But I did change the dispatch to go through the facade